subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,66 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more and more popular. Are you wondering why? Because a
+subscription model simplifies the business process and makes it very easy for both customers and the
+business. It ensures that the business retains more customers on a consistent basis and that clients
+can be reached out to and engaged with regularly. A business model like this makes it easy to
+anticipate demand and supply. Finally, this model offers much higher payment security for your
+business.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
+   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
+   management.</b> </div>
+
+Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
+help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
+possibility to create, edit and manage your own subscription templates.
```

Comment:
```suggestion
Using templates allows you to save time as they allow you to generate recurring invoices and manage renewals at a glance, and keep consistency, as you do not need to create individual forms for every transaction.
```

It has to be improved of course, but you might get my suggestion. A suggestion that could be used as a base to a brainstorm (: